project/recipes_api.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import os
+
+def get_recipes_from_api(ingredients):
+    api_url = "https://api.spoonacular.com/recipes/findByIngredients"
+    api_key = os.getenv("SPOONACULAR_API_KEY") 
+    params = {
+        'apiKey': api_key,
+        'ingredients': ingredients,
+        'number': 5,  # Number of recipes to return
+        'ranking': 1,
+        'ignorePantry': True
+    }
+    response = requests.get(api_url, params=params)
+    if response.status_code == 200:
+        return response.json()
+    else:
+        print(f"Error: {response.status_code}")
```

Comment:
Printing will not work as expected on a server. This is something I haven't figured out how to do yet on flask so it's fine for the moment.